internal/handlers/handlers.go

```diff
@@ -59,6 +61,13 @@ func GetLotteryNumbers(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// VersionHandler writes the current application version to the response.
+func VersionHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write([]byte(version.Version))
```

Comment:
It's good practice to handle the error returned by the `Write` function. If there's an error while writing the response, it would be helpful to log it for debugging purposes. [medium]